etc/Dockerfile.cache

```diff
@@ -3,7 +3,14 @@ ARG BASE_TAG=amd64
 
 FROM $MAIN_TAG:$BASE_TAG as builder
 COPY --chown=1000:1000 ./ /tmp/rdk
-RUN --mount=type=secret,id=netrc,uid=1000,dst=/home/testbot/.netrc sudo -Hu testbot bash -lc 'cd /tmp/rdk && make build-go lint-go'
+WORKDIR /tmp/rdk
+RUN --mount=type=secret,id=netrc,uid=1000,dst=/home/testbot/.netrc sudo -Hu testbot bash -lc 'if [ `dpkg --print-architecture` = armhf ]; then \
+        go mod download \
+        && GOFLAGS=-tags=no_tflite go build ./web/cmd/server \
```

Comment:
Is there a reason the architecture detection can't just be in the makefile and then "build-go lint-go" just does the right thing?

Reply:
Still wondering on this. Why not just `touch web/runtime-shared/static/control.js && make server lint-go tool-install` here for all cases?

Reply:
the `no_tflite` tag is designed for the registration pattern in our codebase, but the lint-go command uses `go list` in a way that circumvents the protections

I pushed a change to make it `make build-go tool-install`

can look into the lint command if you truly want these to be the same. `go list` does respect build tags, this could be made to work probably